torch/csrc/jit/passes/python_print.cpp

```diff
@@ -834,6 +835,19 @@ struct PythonPrintImpl {
         }
         ss << "]";
       }
+    } else if (v.isTuple()) {
+      const auto& elems = v.toTuple()->elements();
+      ss << "(";
```

Comment:
Can we deduplicate the logic for printing a tuple with the TupleConstruct printing?